packages/frontend/src/pages/timeline.vue

```diff
@@ -83,35 +83,62 @@ function top(): void {
 
 async function chooseList(ev: MouseEvent): Promise<void> {
 	const lists = await userListsCache.fetch();
-	const items = lists.map(list => ({
-		type: 'link' as const,
-		text: list.name,
-		to: `/timeline/list/${list.id}`,
-	}));
+	const items = [
+		...lists.map(list => ({
+			type: 'link' as const,
+			text: list.name,
+			to: `/timeline/list/${list.id}`,
+		})),
+		null,
```

Comment:
要素がないときは仕切りを表示しないようにしたほうが見た目が良さそうな気がします（ノートのメニュー内の「クリップ」はそうはなっていませんが…）。
```suggestion
		(lists.length === 0 ? undefined : null),
```

Reply:
ありがとうございます！